test/web/controllers/questionnaire_page_test.exs

```diff
@@ -22,7 +22,7 @@ defmodule Advisor.Web.RequestPageTest do
     # maybe split this later into group_lead and advisor
     assert response
              |> Floki.find(".people-picker li")
-             |> length == 29
+             |> length == 27
```

Comment:
There is double pain here:
1. magic numbers
2. dependence on the li and double-counting. 

You could make the css more robust and check against a count from the DB-1?